app/components/modals/event-delete-modal.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+import ModalBase from 'open-event-frontend/components/modals/modal-base';
+
+const { computed } = Ember;
+
+export default ModalBase.extend({
+  isSmall       : true,
+  confirmName   : '',
+  eventName     : '',
+  doesNameMatch : computed('confirmName', function() {
```

Comment:
The variable name `doesNameMatch` is confusing. Currently, if `doesNameMatch=true` then that actually means they don't match.